web/webpack.config.js

```diff
@@ -4,7 +4,10 @@ const { composePlugins, withNx } = require("@nx/webpack");
 const { withReact } = require("@nx/react");
 const { merge } = require("webpack-merge");
 
-require("dotenv").config();
+require("dotenv").config({
+  // resolve the .env file in the root of the project ../
+  path: path.resolve(__dirname, "../.env"),
```

Comment:
what happens if .env folder doesn't exist??

Reply:
If the file doesn't exist, it just doesn't read it.